agriculture_shop/static/src/js/my_tour.js

```diff
@@ -0,0 +1,17 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("knowledge_tour", {
+    url: "/odoo",
+    
+    steps: () => [
+        {
+            trigger: '.o_app[data-menu-xmlid="knowledge.knowledge_menu_root"]',
+            content: _t("Open Knowledge to find the onboarding guide."),
```

Comment:
For each module, update content of tour:
content: _t("Get on track and explore our recommendations for your Odoo usage here!"),